tests/umem_tests.rs

```diff
@@ -73,7 +73,7 @@ async fn shared_umem_returns_new_fq_and_cq_when_sockets_are_bound_to_different_d
         .unwrap();
 }
 
-#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+/* #[tokio::test(flavor = "multi_thread", worker_threads = 2)]
```

Comment:
This test segfaults.  I think it has to do with the Drop that happens when the sockets go out of scope.  The first one removes the xdp program, and the second tries as well and fails.